packages/web/src/scss/tools/__tests__/_accessibility.test.scss

```diff
@@ -0,0 +1,82 @@
+// stylelint-disable scss/at-mixin-argumentless-call-parentheses -- We need to allow this to make sass-true work
+@use 'true';
+@use '../accessibility';
+
+@include true.describe('hide-text mixin') {
+    @include true.it('outputs styles to visually hide text') {
```

Comment:
Please, use `should` word to start the sentence of the test description like `it should outputs ...`

Reply:
added